src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -92,6 +92,7 @@ function SidebarLinksData({isFocused, allReportActions, betas, chatReports, curr
             style={[styles.flex1, styles.h100]}
         >
             <SidebarLinks
+                currentReportID={currentReportID}
```

Comment:
Another PR somehow changed this, this call to `getTopmostReportId` was `currentReportID` I believe.

<img width="693" alt="CleanShot 2023-09-14 at 15 45 13" src="https://github.com/Expensify/App/assets/1634213/ce4703d0-1e03-4b67-91bf-9a7bcf1f0bfa">

I will remove it but it seems a lot of people want to change this somehow. On the next PR Janic also suggested replacing this with a call to Navigation. TBH I don't know which one is correct since I'm not familiar with all the pieces of the app.